src/lib/holocene/calendar.svelte

```diff
@@ -66,11 +66,17 @@
   }
 
   .cell {
-    @apply m-1 inline-flex h-[24px] w-[24px] items-center justify-center rounded p-1 text-sm;
+    @apply m-1 inline-flex h-[24px] w-[24px] items-center justify-center rounded p-1 text-sm 
+    hover:surface-interactive
+    active:surface-interactive  
+    hover:rounded-2xl 
+    hover:text-inverse 
+    active:text-inverse;
```

Comment:
I think we want to move these to the .highlight class. Otherwise the days of the week ("Su","Mo"...) and the days that are not of the current month get this background too. We just want the cells you can select to have the background. I'd also probably use bg-interactive and text-offWhite since you are overriding the surface text color anyways.

Reply:
Thank you so much, this is very helpful.